src/osd/ReplicatedPG.cc

```diff
@@ -7075,7 +7075,17 @@ int ReplicatedPG::fill_in_copy_get(
       cursor.data_complete = true;
       dout(20) << " got data" << dendl;
     }
-    assert(cursor.data_offset <= oi.size);
```

Comment:
I don't follow how this can happen.  Isn't data_offset always 0 or a result of a previous copy_get?